common/src/thrift/file_system_master.thrift

```diff
@@ -42,6 +43,16 @@ struct MkdirTOptions {
   3: optional bool allowExists
 }
 
+struct PersistCommand {
+  1: common.CommandType commandType
```

Comment:
It's pretty confusing to get a `PersistCommand` telling you to delete or free something. I'd prefer to change the struct name to something like FileSystemMasterCommand.